test/postgresql.autoupdate.test.js

```diff
@@ -643,6 +552,7 @@ describe('autoupdate', function() {
         }
 
         // do initial update/creation of table with fk
+        ds.createModel(product_schema.name, product_schema.properties, product_schema.options);
```

Comment:
Thank you for providing the test flow, it's very helpful for reviewer. Could you also include it in the code comment? 

Like IIUC here should be:
Model Order has a foreign key customerId refers Model C2
Initial update/creation of table with fk

Reply:
Added